lib/webResource.ts

```diff
@@ -231,6 +231,18 @@ export class WebResource {
 
     return this;
   }
+
+  /**
+   * Clone this WebResource HTTP request object.
+   * @returns {WebResource} The clone of this WebResource HTTP request object.
+   */
+  clone(): WebResource {
+    const result = new WebResource(this.url, this.method, this.body, this.query, this.headers, this.rawResponse, this.abortSignal);
+    result.formData = this.formData;
+    result.operationSpec = this.operationSpec;
+    result.abortSignal = this.abortSignal;
```

Comment:
redundant?

Reply:
Removed in https://github.com/Azure/ms-rest-js/pull/100/commits/5fd6b70d34d2521918ec6630e02a13288f41c69f.